github/create-release.yml

```diff
@@ -28,41 +28,20 @@ steps:
     RELEASE_TITLE: ${{ parameters.releaseTitle }}
     RELEASE_NOTES: ${{ parameters.releaseNotes }}
   displayName: Check for required parameters in YAML template
-- powershell: |
-    if ($Env:TAG -match '-') {
-      Write-Host "GitHub release is pre-release because of $Env:TAG"
-      Write-Host "##vso[task.setvariable variable=isPreRelease]true"
-      Write-Host "##vso[task.setvariable variable=compareWith]lastRelease"
-      Write-Host "##vso[task.setvariable variable=releaseNotes]$Env:RELEASE_NOTES"
-      Write-Host "##vso[task.setvariable variable=isDraft]false"
-    } else {
-      Write-Host "GitHub release is full-release because of $Env:TAG"
-      Write-Host "##vso[task.setvariable variable=isPreRelease]false"
-      Write-Host "##vso[task.setvariable variable=compareWith]lastFullRelease"
-
-      $releaseSummary = 
-    @"
-    
-    ## What's new?
-    ### Features
-
-    ### Fixes / Changes
-    None.
-
-    ### Breaking Changes
-    None.
-
-    ### Removal
-    None.
-    "@
-     
-      $releaseNotes = $Env:RELEASE_NOTES + $releaseSummary
-      Write-Host "##vso[task.setvariable variable=releaseNotes]$releaseNotes"
-      Write-Host "##vso[task.setvariable variable=isDraft]true"
-    }
+- bash: |
```

Comment:
Because it first seemed that bash would support multi-line variables with its `echo` command, but it turns out that it also doesn't. Changed it so that the parameter value is used directly (which is a multi-line) which is also already set in every repository.

Reply:
We could change it back to PS, though. Maybe we should.

Reply:
Reverted it back https://github.com/arcus-azure/azure-devops-templates/pull/54/commits/9ea772c290d19f5fd7099644d40b943de7fdd352